src/main/java/edu/kis/powp/jobs2d/events/SelectLoadMacroOptionListener.java

```diff
@@ -1,21 +1,19 @@
 package edu.kis.powp.jobs2d.events;
 
-import edu.kis.powp.jobs2d.command.DriverCommand;
 import edu.kis.powp.jobs2d.command.manager.DriverCommandManager;
+import edu.kis.powp.jobs2d.drivers.MacroDriver;
 import edu.kis.powp.jobs2d.features.CommandsFeature;
 import edu.kis.powp.jobs2d.features.MacroFeature;
 
 import java.awt.event.ActionEvent;
 import java.awt.event.ActionListener;
-import java.util.List;
 
 public class SelectLoadMacroOptionListener implements ActionListener {
+    MacroDriver macroDriver = MacroFeature.getMacroDriver();
+    DriverCommandManager manager = CommandsFeature.getDriverCommandManager();
```

Comment:
you haven't removed the dependencies (imports are still there). Please use dependency injection (by constructor)